docs/sources/tutorials/send-metrics-to-prometheus.md

```diff
@@ -0,0 +1,154 @@
+
+It also creates a [prometheus.scrape][https://grafana.com/docs/alloy/latest/reference/components/prometheus.scrape/] component named `"scrape_metrics”` which does the following:
+1. It connects to the `local_system` component (its "source" or target).
+1. It forwards the metrics it scrapes to the "receiver" of another component called `filter_metrics` which you will define next.
+1. It tells Alloy to scrape metrics every 10 seconds. 
+
+### Second Component: Filter metrics
+
+Filtering non-essential metrics before sending them to a data source can help you reduce costs and enable you to focus on the data that matters most. The filtering strategy of each organization will differ as they have different monitoring needs and setups. 
+
+The following example demonstrates filtering out or dropping metrics before sending them to Prometheus. 
+
+Paste this component next in your configuration file:
+```alloy
+prometheus.relabel "filter_metrics" {
+   rule {
+    action = "drop"
+    source_labels =[ "env"]
+    regex = "dev"
+  }
+ forward_to = [prometheus.remote_write.metrics_service.receiver]
+}
+```
+1. `prometheus.relabel` is a component most commonly used to filter Prometheus metrics or standardize the label set passed to one or more downstream receivers. 
+1. In this example, you create a `prometheus.relabel` component named “filter_metrics”. 
+  This component receives scraped metrics from the `scrape_metrics` component you created in the previous step. 
+1. There are many ways to [process metrics](https://grafana.com/docs/alloy/latest/reference/components/prometheus.relabel/). 
+  Within this component, you can define rule block(s) to specify how you would like to process metrics before they are stored or forwarded. 
+1. This example assumes that you are monitoring a production environment and the metrics collected from the dev environment will not be needed for this particular use case. 
+1. To instruct Alloy to drop metrics whose environment label, `[”env]”`, is equal to `”dev”`, you set the `action` parameter to `”drop”`, set the `source_labels` parameter equal to `[“env”]`, and the `regex` parameter to `“dev”`.  
+1. You use the `forward_to` parameter to specify where to send the processed metrics. In this case, you will send the processed metrics to a component you will create next called `metrics_service`. 
+
+### Third Component: Write metrics to Prometheus
+
+Paste this component next in your configuration file:
+```alloy
+
+prometheus.remote_write "metrics_service" {
+    endpoint {
+        url = "http://localhost:9090/api/v1/write"
+
+        //basic_auth {
+            //username = "admin"
+            //password = "admin"
+      // }
+    }
+}
+
+```
+This last component creates a [prometheus.remote_write][https://grafana.com/docs/alloy/latest/reference/components/prometheus.remote_write/] component named `metrics_service` that points to `http://localhost:9090/api/v1/write`.
```

Comment:
````suggestion
```
This last component creates a [prometheus.remote_write][prometheus.remote_write] component named `metrics_service` that points to `http://localhost:9090/api/v1/write`.
````